utils/retro_wrappers.py

```diff
@@ -339,15 +339,19 @@ def step(self, action):
 
 class MultipleStates(gym.Wrapper):
     """This wrapper randomly loads a state listed in the state_names
-    upon reset"""
-    def __init__(self, env, state_names = -1):
+    upon reset after max_count episodes"""
+    def __init__(self, env, state_names = -1, max_count = 5):
         gym.Wrapper.__init__(self, env)
         self.state_names = state_names
+        self.max_count = max_count
+        self.count = 0
         
     def reset(self):
-        if type(self.state_names) == list:
+        self.count += 1
+        if type(self.state_names) == list and self.count > self.max_count:
```

Comment:
Aqui é pra ser ```self.count > self.max_count``` ou ```self.count >= self.max_count```?

Reply:
Levei em conta que no início de cada episódio damos `env.reset()`, então `self.count` já começa valendo 1. Por isso, optei por `self.count > self.max_count`, para ter realmente `max_count` episódios.

Reply:
então o ```self.count``` não deveria resetar para 1 na linha aqui debaixo dentro do ```reset()```?

Reply:
Sim, esqueci de mudar

Reply:
Já arrumei